### DISCUSSION TITLE
Origami & Chill

### DISCUSSION THREAD

**Discussion by 7368697661** [7368697661/Origami#19] [Announcements]
# Origami Workshop

hey everyone! i just wanted to start a few discussion boards for a few theme topics. if anyone has any feedback or feature requests, i would love to hear them. i can't promise i will be the quickest, but I will try to add stuff with enough interest :)

- are there any parts of origami that you like?
- any parts you wish were different?
- any specific style settings features?
- or maybe colorschemes?

**Comment by irisfish531** [comment]
I love this theme and thank you very much!

**Comment by Emileemandee** [comment]
The theme is absolutely beautiful! I would just love to have an option to adjust the header size in the style settings.
